NSS-Therapeutic-Raccoons/WorkforceManagement
# Rbmodels

## Link to Ticket
Closes # [17](https://github.com/NSS-Therapeutic-Raccoons/WorkforceManagement/issues/17)

## Description of Proposed Changes
- Created Models, PR Template, updated .gitignore, and updates SQL file.

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout rbmodels
```

look at the files, verify the stuff is there. BOOM.


## Impacted Areas in Application

List general components of the application that this PR will affect:

* Everything.

## Mentions @username

@khardt
@figamus
@jeremiah3643
@ricky-bruner
@askingalot

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
